cherryontech/jupiter-jumpers
# Adding project scaffolding

## Description
- Initialized next.js project 
- Added tailwind, typescript, prettier
- Updated readme to include scripts 
- Ran prettier scripts to make styling consistent across codebase 


## Type of Changes
- Feature update 


## Acceptance Criteria
- Next.js, tailwind, prettier, typescript all added to project 


## Update Screenshots
- N/A


## Testing Instructions
1. Fetch latest changes 
2. Run scripts locally to ensure they work 


## Learnings (Optional)
- N/A
